packages/react-focus/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "@fluentui/react-focus",
+  "version": "0.0.1",
+  "description": "Focus helpers to be used in React applications.",
+  "private": true,
+  "main": "lib-commonjs/index.js",
+  "module": "lib/index.js",
+  "typings": "lib/index.d.ts",
+  "sideEffects": [
+    "lib/version.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/fluent-ui-react"
+  },
+  "license": "MIT",
+  "scripts": {
+    "build": "just-scripts build",
+    "clean": "just-scripts clean",
+    "code-style": "just-scripts code-style",
+    "start": "just-scripts dev",
+    "start-test": "just-scripts jest-watch",
+    "update-snapshots": "just-scripts jest -u",
+    "just": "just-scripts"
+  },
+  "devDependencies": {
+    "@types/enzyme": "3.1.13",
+    "@types/enzyme-adapter-react-16": "1.0.3",
+    "@types/es6-promise": "0.0.32",
+    "@types/jest": "23.0.0",
+    "@types/react": "16.8.11",
+    "@types/react-dom": "16.8.4",
+    "@types/react-test-renderer": "^16.0.0",
+    "@types/webpack-env": "1.13.9",
+    "@uifabric/build": "^7.0.0",
+    "@uifabric/example-app-base": "^7.11.1",
+    "@uifabric/jest-serializer-merge-styles": "^7.0.6",
+    "@uifabric/prettier-rules": "^7.0.4",
+    "@uifabric/tslint-rules": "^7.0.5",
+    "enzyme": "~3.9.0",
+    "enzyme-adapter-react-16": "^1.2.0",
+    "es6-weak-map": "^2.0.2",
+    "office-ui-fabric-react": "^7.81.0",
```

Comment:
Should this be depending on OUFR at all? Will that create a circular link in graphs?

Reply:
no it should not, but i'm a little unsure about the docs solution.

The docs refer to components within oufr. So i will need to figure out a way to address keeping the examples somewhere else.

Reply:
Yeah we should potentially consider moving the examples to fabric-website or maybe even a dedicated examples package. That would fix a substantial number of circular dep issues.

Reply:
+1 for separating examples

Reply:
Left the examples in OUFR for now to avoid the circular dep and will move elsewhere long-term